beancount/query/query_compile.py

```diff
@@ -204,13 +215,13 @@ def __init__(self, left, right):
 class EvalAdd(EvalBinaryOp):
 
     def __init__(self, left, right):
-        f = lambda x, y: Decimal(x + y)
+        f = lambda x, y: (x + y)
         super().__init__(f, left, right, Decimal)
```

Comment:
With the above change this could simply be `super().__init__(operator.add, left, right, Decimal)` which can also be slightly faster.

Reply:
Good point. I made that change in my latest.